### ISSUE TITLE
process_test_results doesn't work for multi-module builds

### ISSUE THREAD

**Issue by rkennedy-mode** [CircleCI-Public/maven-orb#9]
**Orb Version**

1.0.1

**Describe the bug**

The `process_test_results` command does not work with multi-module project. It assumes a single module and doesn't recursively check for `target/surefire-reports` directories.

**To Reproduce**

Create a Maven multi-module project. Add tests to the sub-module(s). Use `maven/test` to build the project. The "Uploading test results" step will succeed (green check mark), however the following error will appear in the step output:

```
Unable to save test results from /root/project/target/surefire-reports
Error path is not valid /root/project/target/surefire-reports: error accessing path: /root/project/target/surefire-reports: lstat /root/project/target/surefire-reports: no such file or directory
Found no test results, skipping
```

**Expected behavior**

All `target/surefire-reports` directory outputs are aggregated and sent to `store_test_results`.

**Additional context**

None.

**Comment by dsayling** [comment]
@rkennedy-mode - we believe this was resolved in #17 -  please let us know if you have further issues

**Comment by rkennedy-mode** [comment]
Ah, fabulous timing. I'm refactoring our `config.yml` at this very moment. Let me give it a go and I'll report back here.
